ros2cli/ros2cli/node/network_aware.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
```

Comment:
I guess you moved the code without editing it, if that's the case that's okay.

Reply:
I did.